### PR TITLE
Transition W3C Trace Context Test Service to Vanilla PHP

### PR DESCRIPTION
**Pull request by yuktea**
## What
Transitions the w3c-trace-context test to vanilla PHP.

## Why
Speed and simplicity.

### CODE REVIEW

**Comment by bobstrecansky** [comment]
@yuktea - #686 has been merged.  Does that unblock you on this PR?

**Comment by yuktea** [comment]
@bobstrecansky yes it does!

**Comment by yuktea** [comment]
Ready for review

/cc @tidal @Nevay

File: tests/TraceContext/W3CTestService/trace-context-test.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -x
+
+function install_symfony() {
+    curl -1sLf 'https://dl.cloudsmith.io/public/symfony/stable/setup.alpine.sh' | bash >/dev/null
```
Comment (bobstrecansky) [inline tests/TraceContext/W3CTestService/trace-context-test.sh:5]: I'm not familiar with `dl.cloudsmith.io` - downloading from a 3rd party script source feels dangerous to me.  Could we potentially have this script be in our repo?

Comment (yuktea) [reply tests/TraceContext/W3CTestService/trace-context-test.sh:5]: Looks like it's officially supported by Symfony, I think they're using the platform for artifact management beyond GitHub: https://symfony.com/download ; https://cloudsmith.com/

File: Makefile [hunk 2]
```diff
@@ -63,7 +63,7 @@ style:
 deptrac:
 	$(DC_RUN_PHP) env XDEBUG_MODE=off vendor/bin/deptrac --formatter=table --report-uncovered
 w3c-test-service:
-	@docker-compose -f docker-compose.w3cTraceContext.yaml run --rm php ./tests/TraceContext/W3CTestService/symfony-setup
+	@docker-compose -f docker-compose.w3cTraceContext.yaml run --rm php ./tests/TraceContext/W3CTestService/trace-context-test.sh
```
Comment (Nevay) [inline Makefile:66]: File is missing executable permission.

Comment (yuktea) [reply Makefile:66]: resolved, thank you

File: tests/TraceContext/W3CTestService/trace-context-test.sh (hunk 1 above)
Comment (bobstrecansky) [reply tests/TraceContext/W3CTestService/trace-context-test.sh:5]: Good enough for me

**Review by bobstrecansky — APPROVED** [review]

**Pull request merged**